scripts/packages/postinstall.sh

```diff
@@ -39,7 +40,18 @@ detect_nginx_users() {
 
     if [ -z "${nginx_user}" ]; then
         printf "PostInstall: Reading NGINX process information to determine NGINX user\n"
-        nginx_user=$(ps aux | grep "nginx: master process" | grep -v grep | head -1 | awk '{print $1}')
+        nginx_pid=""
+        for pid in $(ls /proc | grep -E '^[0-9]+$'); do
```

Comment:
Remember to install the Shellcheck extension for VS Code: https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck

My Shellcheck is complaining about this `ls` command, here's the error it refers to: https://www.shellcheck.net/wiki/SC2010